recipes/scikit-cycling/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "scikit-cycling" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "3acaed9ccfdfa34f43e75342bd86c18bb3ad4b940a24eedb78f674364fba1076" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/scikit-cycling/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export CC=clang  # [osx]
+    - export CXX=clang++  # [osx]
+    - python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain
+    - cython
+    - numpy 1.9.*  # [not (win and py36)]
+    - numpy 1.11.*  # [win and py36]
+    - clangdev
+    - clangdev
```

Comment:
This line is repeated.

Also investigate if it is really needed.

Reply:
without it, it will throw a `-fopenmp` not recognized option by the compiler